apps/fluent-tester/src/RNTester/TestComponents/Checkbox/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const HOMEPAGE_CHECKBOX_BUTTON = 'Homepage_Checkbox_Button';
+export const CHECKBOX_TESTPAGE = 'Checkbox_TestPage';
```

Comment:
nit does our style say to end w/ new lines?

Reply:
Yes it seems so, I just looked around the repo. I will add the new lines back in.